docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TimeSeries/DetectSeasonality.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML;
+using Microsoft.ML.TimeSeries;
+
+namespace Samples.Dynamic
+{
+    public static class DetectSeasonality
+    {
+        public static void Example()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for
+            // exception tracking and logging, as well as the source of randomness.
+            var mlContext = new MLContext();
+
+            var dataView = mlContext.Data.LoadFromEnumerable(GetPointsWithSeasonality());
+            // Create a seasonal data as input for DetectSeasonality.
+            var input = GetPointsWithSeasonality();
+            SeasonalityDetector seasonalityDetector = new SeasonalityDetector();
```

Comment:
I think the general pattern is to add public-facing functionality to one of the extension catalogs